github/table_github_team_member.go

```diff
@@ -0,0 +1,146 @@
+package github
+
+import (
+	"context"
+
+	"github.com/google/go-github/v33/github"
+
+	pb "github.com/turbot/steampipe-plugin-sdk/v3/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v3/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v3/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func gitHubTeamMemberColumns() []*plugin.Column {
+	repoColumns := gitHubUserColumns()
+	teamColumns := []*plugin.Column{
+		{Name: "organization", Type: pb.ColumnType_STRING, Description: "The organization the team is associated with.", Transform: transform.FromQual("organization")},
+		{Name: "slug", Type: pb.ColumnType_STRING, Description: "The team slug name.", Transform: transform.FromQual("slug")},
+		{Name: "role", Type: pb.ColumnType_STRING, Description: "The team member's role", Hydrate: tableGitHubTeamMemberGet},
+		{Name: "state", Type: pb.ColumnType_STRING, Description: "The membership state", Hydrate: tableGitHubTeamMemberGet},
+	}
+
+	return append(repoColumns, teamColumns...)
+}
+
+func tableGitHubTeamMember() *plugin.Table {
+	return &plugin.Table{
+		Name:        "github_team_member",
```

Comment:
Terraform calls this `github_team_members` (plural), but given that everything in singular, it would make more sense to keep it that way IMHO.

Reply:
Thanks @japborst for sticking to the singular naming convention as per our [docs](https://steampipe.io/docs/develop/standards#naming). `github_team_member` is correct 👍